### DISCUSSION TITLE
Does this fork support the API changes that happened in Sept 2024?

### DISCUSSION THREAD

**Discussion by codersaur** [ddvk/rmapi#2] [General]
So I found this fork after the original project stopped working around 10 days ago. See: https://www.reddit.com/r/RemarkableTablet/comments/1fj86xz/has_remarkable_broken_their_cloud_api/

Can you confirm if this fork supports the recent changes and do you plan to maintain this fork?

[EDIT] I tested myself and it seems to be working for 'find' and 'put' requests at least.

Thanks ddvk!

**Comment by ddvk** [comment]
yes, this supports the latest changes

**Reply by rorycl** [reply]
It is working great for me. Thanks *a lot* @ddvk

**Comment by codersaur** [comment]
Funnily enough, it was working great for me up until a few days ago. but now file uploads are returning "request failed with status 202". There must have been another change in the last week.

Update: I see there is already a commit to fix this. Good work ddvk!
